src/Microsoft.DotNet.TestFramework/TestAssets.cs

```diff
@@ -34,7 +56,48 @@ public TestAssetInfo Get(string kind, string name)
         {
             var assetDirectory = new DirectoryInfo(Path.Combine(_root.FullName, kind, name));
 
-            return new TestAssetInfo(assetDirectory, name);
+            return new TestAssetInfo(
+                assetDirectory, 
+                name, 
+                _dotnetCsprojExe,
+                CsprojSearchPattern);
+        }
+
+        public TestAssetInfo GetPJ(string name)
```

Comment:
Well, not a fan of these abbreviations, to be honest. I would call it GetProjectJson.